docs/introduction.md

```diff
@@ -85,3 +85,72 @@ Available download clients and actions
 The development of autobrr started in Early 2020, entering rapid development in Summer 2021 due to dissatisfaction with needing 3+ tools to do one job. Autobrr has since gained quite a bit of traction and has a growing community supporting the project.
 
 Autobrr was developed with resource consumption in mind. The software uses API calls to reduce unnecessary downloads of .torrent files from sites like BTN, RED, PTP, and GGn. On other sites, it will download the .torrent only if the information is not present in the announce message.
+
+## Quick Start
+
+The following considers a common use case, feeding IRC announcements for a private
+tracker to a Servarr instance, as a way to illustrate how the components of autobrr fit
+together and what is required to get up and running.  It's not the only use case and
+other use cases may require more RTFM'ing in [the configuration docs](./configuration).
+
+1. [Install autobrr](./installation/linux).
+
+	Proceed once the web UI is accessible.
+
+2. Create a user for yourself.
+
+	Most easily done through the initial "wizzard" in the web UI when autobrr is first
+	started.
+
+3. [Register a nick on your indexer's IRC
+   network](https://autobrr.com/configuration/irc#registering-with-nickserv).
+
+4. [Group a "bot" nick with your real nick](./configuration/irc#grouping-nicks)
```

Comment:
A note to check your trackers IRC guide so they don't forget any invite commands or grouping nomenclature (|bot or |autodl)